zad_94/js/scripts.js

```diff
@@ -0,0 +1,57 @@
+var poziomy = 0;
+
+function drawTree(poziomy) {   
+    for (var i = 0; i < poziomy; i++) {
+        star = '';
+        for (var j = 0; j <= i; j++) {
+            star +='*';
+        }
+        console.log(star);
+    }
+}
+// wywołanie funkcji drawTree 
+drawTree(5);
+drawTree(10);
+drawTree(6);
+
+// wersja druga z pionowa choinka
+var poziomy1 = 0;
+var pion1 = '.';
```

Comment:
jak usunę var pion1
to ma bład
scripts.js:33 Uncaught ReferenceError: pion1 is not defined
    at drawTree1 (scripts.js:33)
    at scripts.js:50